cpp/log/tree_signer_test.cc

```diff
@@ -171,6 +171,7 @@ TYPED_TEST(TreeSignerTest, Sign) {
   this->test_signer_.CreateUnique(&logged_cert);
   this->AddPendingEntry(&logged_cert);
   // this->AddSequencedEntry(&logged_cert, 0);
+  EXPECT_TRUE(this->tree_signer_->SequenceNewEntries().ok());
```

Comment:
This would be nicer like this (you see the status if it's not OK):

    EXPECT_EQ(util::Status::OK, this->tree_signer_->SequenceNewEntries());